src/render-html/css.js

```diff
@@ -277,6 +277,10 @@ ul {
   margin: 0.25em 0;
   font-size: 10px;
 }
+div.twitter-image > a > img {
```

Comment:
Try to match what the web version is doing more closely.
You don't need the `div` part when it is followed by a class.
`.twitter-image img` is enough.